modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,102 @@
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-46684[MB-46684^]
+| *Summary*: Fixed an issue where the Analytics service rebalance failed because of a Null Pointer Exception.
+
+| https://issues.couchbase.com/browse/MB-44875[MB-44875^]
+| *Summary*: Fixed an issue where the Analytics remote link ingestion rate was much slower compared to local link.
+|===
+
+==== Cross Data Center Replication (XDCR)
+
+[#table_fixedissues_v663-xdcr,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47156[MB-47156^]
+| *Summary*: The XDCR internal setting `health_check_interval` can now be configured the key `HealthCheckIntervalSec`. For information on how to configure internal settings, see xref:rest-api:rest-get-internal-setting.adoc#changing-internal-settings[Changing Internal Settings].
+
+| https://issues.couchbase.com/browse/MB-46760[MB-46760^]
+| *Summary*: Previously, when you created replications, XDCR tried to validate compression with both source and target nodes. This was to ensure that XDCR could successfully negotiate snappy compression between them if compression was expected over the replication pipeline. This validation logged warnings when snappy compression could not be negotiated, for example when replicating from versions earlier than 5.5 to versions 5.5+. 
+
+Starting with version 7.0, this validation of snappy compression is disabled as all active and supported versions of Couchbase Server support snappy compression.
+|===
+
+==== Eventing Service
+
+[#table_fixedissues_v663-eventing,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-45359[MB-45359^]
+| *Summary*: Previously, to debug an Eventing function (via NAT) on a remote Couchbase server, the Eventing debugger needed manual URL edits to connect to the remote cluster. This has been fixed and the Eventing debugger can now use a secondary IP address if available.
+|===
+
+==== Index Service and Views
+
+[#table_fixedissues_v663-gsi-views,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47094[MB-47094^]
+| *Summary*: When request auditing was enabled in a Couchbase cluster, under very high Views query workload, the view-engine audit message queue would grow unbounded causing the view-engine to crash. This has been fixed.
+
+| https://issues.couchbase.com/browse/MB-46187[MB-46187^]
+| *Summary*: Fixed a leak in the singletonWorkers that was causing high CPU utilization by the Index service.
+|===
+ 
+==== Query Service
+
+[#table_fixedissues_v663-query,cols="25,66"]
+|===
+| Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47722[MB-47722^]
+| *Summary*: Fixed a connection leak in go-couchbase by adding timeouts to REST API calls over HTTPS.
+
+| https://issues.couchbase.com/browse/MB-46613[MB-46613^]
+| *Summary*: Fixed an issue where an empty index span was generated for LEFT OUTER JOINs when the WHERE clause was MISSING.
```

Comment:
Should be : 
Fixed an issue where an empty index span was generated for LEFT OUTER JOINs when the WHERE clause has the IS MISSING operator

Reply:
Thanks Mihir. Updated.